benchmarks/simple.mjs

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env node
+
+// Benchmark results are unstable. To have more stable results:
+// 1. Restart OS. Do not run any applications. Put power cable to laptop.
+// 2. Run tests 5 times.
+// 3. Took the best result for each candidate.
+
+import {Bench} from "tinybench"
+import * as colorette from "colorette"
+import kleur from "kleur"
+import * as kleurColors from "kleur/colors"
+import chalk from "chalk"
+import ansi from "ansi-colors"
+import cliColor from "cli-color"
+import picocolors from "../picocolors.js"
+import * as nanocolors from "nanocolors"
+import * as yoctocolors from "yoctocolors"
+
+console.log(colorette.green("colorette"))
+console.log(kleur.green("kleur"))
+console.log(chalk.green("chalk"))
+console.log(ansi.green("ansi"))
+console.log(cliColor.green("cliColor"))
+console.log(picocolors.green("picocolors"))
+console.log(nanocolors.green("nanocolors"))
```

Comment:
**question:** What is this section for?

Reply:
It was in the existing benchmarks. I guess so you can see that each library is actually working when running this